UnityProject/Assets/Scripts/Systems/Explosions/IEMPAble.cs

```diff
@@ -0,0 +1,9 @@
+//Interface is triggered when object is affected by EMP
+
+namespace Systems.Explosions
+{
+	public interface IEMPAble
+	{
+		void OnEMP(int EMPStrength);
```

Comment:
I know the project hasn't been very good at following this, but would be great if you rename this to `IEmpable` and `OnEmp`, `empStrength`.

Reply:
done